editor/resources/bundle.editor_script

```diff
@@ -13,8 +16,8 @@ end)
 
 local function bundle_linux(show_dialog)
     local config = editor.bundle.config(show_dialog, "bundle.linux", linux_bundle_dialog)
-    local output_directory = editor.bundle.output_directory(show_dialog, "x86_64-linux")
-    editor.bundle.create(config, output_directory, {platform = "x86_64-linux"})
+    local output_directory = editor.bundle.output_directory(show_dialog, config.platform)
+    editor.bundle.create(config, output_directory, {platform = config.platform, architectures = config.platform})
```

Comment:
Is it possible that we want to bundle for multiple Linux architectures simultaneously, e.g. like we do for macOS? In that case, we might want to use a set of architecture options instead.

Reply:
While we might, I chose to leave that until we get a request for it.
* macOS is the odd-one-out, as it supports universal binaries.
* win32/win64 bundle flow doesn't support multiple architectures at the same time 

So I modeled it after the win32 flow for now.